### PR TITLE
#160238582 Header card component

### PR DESCRIPTION
**Pull request by veeqtor**
#### What does this PR do?

Header card component

#### Description of Task to be completed?
  - Write unit tests
  - Add semantic-ui CDN
  - Add semantic-ui-react
  - Create the header card component

#### How should this be manually tested?
In ComandLine:
- `git clone https://github.com/andela/fargo-ah-client.git Authors-Haven-Client`
- `npm install`
- `npm run start` the start script
- Go to '/ ' path to check the home route

#### Any background context you want to provide?

N/A

#### What are the relevant pivotal tracker stories?

[#160238582](https://www.pivotaltracker.com/story/show/160238582)

#### Screenshots

screen shot 2018-09-04 at 4 56 18 pm

### CODE REVIEW

File: .stylelintrc [hunk 1]
```diff
@@ -6,4 +6,4 @@
     "indentation": 2,
     "unit-whitelist": ["px", "em", "%", "rem", "ms", "s", "vh"],
   }
-}
\ No newline at end of file
+}
```
Comment (ascii-dev) [inline .stylelintrc]: Newline, please...

**Review by ascii-dev — APPROVED** [review]

**Review by OKiMaureen — APPROVED** [review]

File: src/components/HeaderCard.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { Card, Image, Grid } from 'semantic-ui-react';
+import { Link } from 'react-router-dom';
+import moment from 'moment';
+
+const headerCardFunction = (imageUrl, title, slug, username, authorImage, date) => (
```
Comment (anuonifade) [inline src/components/HeaderCard.js]: Are these supposed to be props?

Comment (veeqtor) [reply src/components/HeaderCard.js]: No, they are arguments passed to the `headerCardFunction` function from the article props

**Review by anuonifade — APPROVED** [review]
LGTM

**Pull request merged**